src/components/_alert.scss

```
@@ -7,104 +7,368 @@ category: Components
+    <img class="ncgr-alert__icon" src="./images/components/alert/alert-icon-white.svg">
+  </div>
+  <div class="ncgr-alert__title">
+    <h4 class="ncgr-alert__primary-title -bold">
+      注意喚起
+    </h4>
+    <div class="ncgr-alert__secondary-title">
+      詳しく見る
+      <i class="fa fa-angle-right ncgr-mar-l-4"></i>
+    </div>
+  </div>
+</a>
+<a class="ncgr-alert -color-danger -appearance-flat">
+  <div class="ncgr-alert__leading">
+    <img class="ncgr-alert__icon" src="./images/components/alert/alert-icon-white.svg">
+  </div>
+  <div class="ncgr-alert__title">
+    <h4 class="ncgr-alert__primary-title -bold">
+      発生中のエラー
+    </h4>
+    <div class="ncgr-alert__secondary-title">
+      詳しく見る
+      <i class="fa fa-angle-right ncgr-mar-l-4"></i>
+    </div>
+  </div>
+</a>
 ```
 
 */
 
+$colors: default, primary, success, warning, danger;
+
 .ncgr-alert {
-  background-color: var(--color-default-50);
+  display: flex;
+  align-items: center;
   border-radius: 8px;
   box-sizing: border-box;
-  color: var(--color-default-900);
-  padding: 16px;
+  padding: 1rem;
   position: relative;
   width: 100%;
-  @media (prefers-color-scheme: dark) {
-    background-color: var(--color-default-200);
-  }
-  &.ncgr-alert--default {
-    background-color: var(--color-default-50);
-    color: var(--color-default-900);
-    @media (prefers-color-scheme: dark) {
-      background-color: var(--color-default-200);
-    }
-  }
-  &.ncgr-alert--primary {
-    background-color: var(--color-primary-50);
-    color: var(--color-primary-900);
-    @media (prefers-color-scheme: dark) {
-      background-color: var(--color-primary-200);
-    }
-  }
-  &.ncgr-alert--success {
-    background-color: var(--color-success-50);
-    color: var(--color-success-900);
-    @media (prefers-color-scheme: dark) {
-      background-color: var(--color-success-200);
-    }
-  }
-  &.ncgr-alert--warning {
-    background-color: var(--color-warning-50);
-    color: var(--color-warning-900);
-    @media (prefers-color-scheme: dark) {
-      background-color: var(--color-warning-200);
-    }
-  }
-  &.ncgr-alert--danger {
-    background-color: var(--color-danger-50);
-    color: var(--color-danger-900);
-    @media (prefers-color-scheme: dark) {
-      background-color: var(--color-danger-200);
+  @each $color in $colors {
+    &.-color-#{$color} {
+      color: var(--color-#{$color}-900);
+      &.-appearance-white {
+        background-color: var(--color-secondary-grouped-background);
+      }
+      &.-appearance-flat {
+        background-color: var(--color-#{$color}-50);
+        @media (prefers-color-scheme: dark) {
+          background-color: var(--color-#{$color}-200);
+        }
+      }
     }
   }
-  .ncgr-alert__heading {
-    font-size: 18px;
-    font-weight: bold;
-  }
-  .ncgr-alert__link{
-    text-decoration: underline;
-    cursor: pointer;
-    color: unset;
+}
+
+a.ncgr-alert {
+  cursor: pointer;
+}
+
+.ncgr-alert__leading {
```

InhouseだとElementにもコンポーネントの名前つけると冗長だしこう書いてるのでこうして入れ子にしとくといいかも

```suggestion
._leading {
```